newIDE/app/src/InstructionOrExpression/EnumerateInstructions.js

```diff
@@ -225,6 +225,13 @@ const enumerateInstruction = (
   ignoresGroups = false
 ): EnumeratedInstructionMetadata => {
   const displayedName = instrMetadata.getFullName();
+  let description = instrMetadata.getDescription();
+  if (description.length > 140) {
```

Comment:
Can we check this is actually worth doing? By looking at the search times, does it change anything?

Reply:
This field can be filled by extension creator so it's a protection against very long description.